### PR TITLE
Updates for release documentation around 1.2

### PR DESCRIPTION
**Pull request by peternied**
### Description
Update release template based on initial changes for the 1.2 release

See the release issue at https://github.com/opensearch-project/opensearch-build/issues/567

### Check List
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 1]
```diff
@@ -18,23 +18,33 @@ I noticed that a manifest was automatically created in [manifests/{{ env.VERSION
...
-### CI/CD
+### CI/CD - _Ends TDB_
 
 - [ ] Create Jenkins workflows that run daily snapshot builds for OpenSearch and OpenSearch Dashboards. 
 - [ ] Increment each component version to {{ env.VERSION }} and ensure working CI in component repositories.
 - [ ] Make pull requests to add each component to [manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml) and [manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml) with the corresponding checks.
 
-### Release
+### Campaigns
+_OpenSearch wide initiatives to improve quailty and consistancy_
```
[the remark this answers is not in the document]
Comment (peternied) [reply .github/ISSUE_TEMPLATE/release_template.md:28]: 🔥 🚑  I'm off to the burn ward!

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 2]
```diff
@@ -44,3 +48,6 @@ I noticed that a manifest was automatically created in [manifests/{{ env.VERSION
 - [ ] Create [release tags](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#tagging) for each component.
 - [ ] Update [this template](./release_template.md) with any new or missed steps.
 - [ ] Conduct a postmortem, and publish its results.
+
+### Components
+__Section for links to all component tracking issues__
```
Comment (dblock) [inline .github/ISSUE_TEMPLATE/release_template.md:53]: Nit: this is `__` vs. Campaigns is `_`, let's pick one?

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 3]
```diff
@@ -25,9 +25,13 @@ I noticed that a manifest was automatically created in [manifests/{{ env.VERSION
...
 
+### Campaigns
+_OpenSearch wide initiatives to improve quality and consistency._
+
 ### Release testing - _Ends RELEASE-minus-6-days_
-- [ ] Code complete phase (RELEASE-minus-14-days - RELEASE-minus-11-days): Teams test their component within the distribution, ensuring integration, backwards compat and perf tests pass.
-- [ ] Final sanity testing phase (RELEASE-minus-8-days - RELEASE-minus-6-days): Final sanity testing *and* fixing of any issues found by teams.
+
+- [ ] Code Complete (RELEASE-minus-14-days - RELEASE-minus-11-days): Teams test their component within the distribution, ensuring integration, backwards compatibility and perf tests pass.
+- [ ] Sanity Testing (RELEASE-minus-8-days - RELEASE-minus-6-days): Sanity testing *and* fixing of any issues found by teams.
```
Comment (dblock) [inline .github/ISSUE_TEMPLATE/release_template.md:34]: any *critical* issues?

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 4]
```diff
@@ -25,9 +25,13 @@ I noticed that a manifest was automatically created in [manifests/{{ env.VERSION
...
 - [ ] Make pull requests to add each component to [manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml) and [manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml) with the corresponding checks.
 
+### Campaigns
+_OpenSearch wide initiatives to improve quality and consistency._
+
 ### Release testing - _Ends RELEASE-minus-6-days_
-- [ ] Code complete phase (RELEASE-minus-14-days - RELEASE-minus-11-days): Teams test their component within the distribution, ensuring integration, backwards compat and perf tests pass.
-- [ ] Final sanity testing phase (RELEASE-minus-8-days - RELEASE-minus-6-days): Final sanity testing *and* fixing of any issues found by teams.
+
+- [ ] Code Complete (RELEASE-minus-14-days - RELEASE-minus-11-days): Teams test their component within the distribution, ensuring integration, backwards compatibility and perf tests pass.
```
Comment (dblock) [inline .github/ISSUE_TEMPLATE/release_template.md:33]: oxford comma?

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 5]
```diff
@@ -25,9 +25,13 @@ I noticed that a manifest was automatically created in [manifests/{{ env.VERSION
 - [ ] Increment each component version to {{ env.VERSION }} and ensure working CI in component repositories.
 - [ ] Make pull requests to add each component to [manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-{{ env.VERSION }}.yml) and [manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml](/opensearch-project/opensearch-build/tree/main/manifests/{{ env.VERSION }}/opensearch-dashboards-{{ env.VERSION }}.yml) with the corresponding checks.
 
+### Campaigns
+_OpenSearch wide initiatives to improve quality and consistency._
```
Comment (dblock) [inline .github/ISSUE_TEMPLATE/release_template.md:29]: Add a space above for consistency with other sections.

File: README.md [hunk 6]
```diff
@@ -73,7 +73,7 @@ The build order first publishes `OpenSearch` followed by `common-utils`, and pub
 
 ##### OpenSearch Dashboards
 
-The build order first pulls down `OpenSearch-Dashboards` and then utilizes it to build other components. Currently, building plugins requires having the core repository built firstt to bootstrap and build the modules utilized by plugins.
+The build order first pulls down `OpenSearch-Dashboards` and then utilizes it to build other components. Currently, building plugins requires having the core repository built first to bootstrap and build the modules utilized by plugins.
```
Comment (dblock) [inline README.md:76]: 🤦  this is me

**Review by dblock — CHANGES_REQUESTED** [review]
Couple more?

File: .github/ISSUE_TEMPLATE/release_template.md [hunk 7]
```diff
@@ -50,4 +53,5 @@ _OpenSearch wide initiatives to improve quality and consistency._
 - [ ] Conduct a postmortem, and publish its results.
 
 ### Components
-__Section for links to all component tracking issues__
\ No newline at end of file
+
+__Replace with links to all component tracking issues__
```
Comment (dblock) [inline .github/ISSUE_TEMPLATE/release_template.md:57]: You're still missing a period! ;)

**Pull request merged**